src/main/resources/com/google/api/codegen/php/client_impl.snip

```diff
@@ -503,13 +503,21 @@
 @private paramList(params)
     @join param : params on ", "
         @if param.defaultValue
-            ${@param.name} = {@param.defaultValue}
+            {@paramName(param)} = {@param.defaultValue}
         @else
-            ${@param.name}
+            {@paramName(param)}
         @end
     @end
 @end
 
+@private paramName(param)
```

Comment:
If this can have a type hint in it, it's not technically just a "name" any more, right?

Reply:
Updated to `paramDeclaration`, WDYT?

Reply:
That works!